### ISSUE TITLE
GUI: Little improvement for list of tables

### ISSUE THREAD

**Issue by Daijobou** [phpmyadmin/phpmyadmin#16864]
See PhpMyAdmin version 5.1.0 in Plesk Obsidian with a single database and many tables.
![image](https://user-images.githubusercontent.com/260343/116369683-eed46100-a809-11eb-8483-698d69f5539f.png)

The last table is not good to read or select. I have this issue in Chrome. This happens when the horizontal scrollbar is displayed. The scroll bar is in the lowest position.

**Comment by 5348** [comment]
try this...use margin bottom style in table tag.
Eg---
<table style=" margin-bottom: 20px;">
  <tr>
    <th>Firstname</th>
    <th>Lastname</th> 
    <th>Age</th>
  </tr>
  <tr>
    <td>Jill</td>
    <td>Smith</td>
    <td>50</td>
  </tr>
  <tr>
    <td>Eve</td>
    <td>Jackson</td>
    <td>94</td>
  </tr>
  <tr>
    <td>John</td>
    <td>Doe</td>
    <td>80</td>
  </tr>
</table>

**Comment by zachweix** [comment]
I think the way to handle this, and in general my suggestion is as follows: The table should be in a div element so that scrolling horizontally or vertically does not move the page, but just the table like the "Fixed Header Table" on https://adminlte.io/themes/v3/pages/tables/simple.html
If you make your window narrow you'll see the horizontal scroll

**Comment by williamdes** [comment]
I can not really confirm the issue but I agree there is not all the space needed when a scroll bar pops out

Can someone confirm that changing
```css
li.last.database {
  margin-bottom: 15px !important;
}
```
to
```css
li.last.database {
  margin-bottom: 25px !important;
}
```
Makes it better ?
(try changing in your browser)

you can search for `#pma_navigation_tree_content>ul>li.last.database` in the browser dom inspector

**Comment by williamdes** [comment]
Last time it was reported as #14898 fixed by #15442

**Comment by williamdes** [comment]
This is how it looks after my fix

Bootstrap theme was really affected by this issue, not the other ones but I synced the rule across all themes anyway

![image](https://user-images.githubusercontent.com/7784660/125622906-b1553c9a-f371-4fa3-be6f-8144e1a29206.png)
![image](https://user-images.githubusercontent.com/7784660/125622962-c44e7c89-e8a7-4026-ad01-164817017806.png)
